services/meta/ota.md

```diff
@@ -7,34 +7,44 @@ This service is used to manage over-the-air (OTA) upgrades of end devices.
 
 #### Interfaces
 
-Type | Interface               | Value type | Description
------|-------------------------|------------|--------------
-out  | evt.ota_progress.report | int_map    | Shows upgrade progress and estimated time left.
-out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade status.
+Type | Interface               | Value type | Description                                                                                                      |
+-----|-------------------------|------------|------------------------------------------------------------------------------------------------------------------|
+in   | cmd.status.get_report   | null       | Requests update progress information.                                                                            |
+out  | evt.status.report       | object     | Shows update progress and estimated time left. See [`ota_report_object`](#definitions) for detailed information. |
+in   | cmd.update.install      | string     | Starts an update of the device. Value should be the full path to the update file.                                | 
 
+## Definitions
+
+* `ota_report_object` is an object with the following structure:
+
+| Field          | Type   | Example         | Description                                                                                                     |
+|----------------|--------|-----------------|-----------------------------------------------------------------------------------------------------------------|
+| state          | string | `"IN PROGRESS"` | One of the `state` values.                                                                                      |
```

Comment:
Wouldn't it be better to declare this field as an enum?

Reply:
It's declared an enum, the list of values is in Definition section. Possibly @PaulinaDrozda you could add a link to the definition section, like:

```
One of the `state` values as defined [here](#definitions)
````